### DISCUSSION TITLE
SUFFIX and PREFIX on generated classes

### DISCUSSION THREAD

**Discussion by marque88** [quarkiverse/quarkus-openapi-generator#465] [Q&A] [answered]
Hi, there is a way to add a custom SUFFIX or PREFIX to generated api and model classes?

Following you documentation I found a solution to customize only the target base-package using

```
quarkus.openapi-generator.codegen.spec.perstore_json.base-package=org.acme.openapi.animals
```

**Comment by hbelmiro** [comment, marked as answer]
There's no way to do that @marque88.
If it's something important to you, please create an issue. And feel free to send a PR.

**Reply by marque88** [reply]
Thanks @hbelmiro
